common/app/views/fragments/nav/editionPicker.scala.html

```diff
@@ -0,0 +1,19 @@
+@()(implicit request: RequestHeader)
+
+@import common.{Edition, LinkTo}
+
+<div class="new-header__edition-picker">
+    @fragments.inlineSvg("world", "icon")
+
+    <select onchange="location = this.value;">
```

Comment:
This won't work via keyboard arrows (I think) because the change event is triggered every time the user activates an option, and in order to activate the third option she has to go through the first and the second.